src/exec.cpp

```diff
@@ -984,6 +987,7 @@ void Executor::raise(const std::string &exception, const std::string &info)
     globals[0].array()[0] = Cell(exception);
     globals[0].array()[1] = Cell(info);
     globals[0].array()[2] = Cell(number_from_uint32(static_cast<uint32_t>(ip)));
+    auto eip = ip;
```

Comment:
Move this down a line (after the blank line) because it relates to the following code, not to the initialization of the exception data.